Makefile

```diff
@@ -171,6 +171,7 @@ docker-tag-release:
 # Push for Dev
 docker-push-dev:  docker-tag-dev
 	@docker push $(REPOSITORY):$(DEV_TAG)
+	@docker push $(REPOSITORY):dev
```

Comment:
Isn't `DEV_TAG` == `dev`
https://github.com/open-policy-agent/gatekeeper/blob/58451d20b124df7f7adc031c3c14e196ab2c6373/Makefile#L6

Reply:
that get's overwritten in https://github.com/open-policy-agent/gatekeeper/blob/master/.github/workflows/workflow.yaml#L108. Default is set in case someone runs it manually without the CI.

Reply:
can you add a comment here

Reply:
added